README.md

```diff
@@ -3,6 +3,5 @@
 A collection of tools that are useful in a git-controlled monorepo that is managed by one of these software:
 
 - rush
-- lerna
```

Comment:
I removed lerna support (since we didn't have lerna support in the stuff we moved over). It's really hard to know how to read the lerna.json anyways. Because every lerna project needs a lerna.json, but most lerna repos do not define workspaces in lerna.json. Instead they use yarn workspaces.

Reply:
lerna also works with npm, though. So I didn't want to exclude those folks!